### ISSUE TITLE
data available ?

### ISSUE THREAD

**Issue by besacier** [andre-pereira/ICMI2024LLMsEnjoymentDetection#1]
hi, i was wondering is the data related to this work will be available soon ? tks for your answer. Laurent

**Comment by andre-pereira** [comment]
Yes, we will add what is possible soon. Some parts of the dataset (shared in https://zenodo.org/records/12588810) are not public (videos and some parts of the text that had to be anonymized). That made us take longer to assess what we can share publicly here on Github and what will only be available only upon request.
